README.md

```diff
@@ -1,47 +1,185 @@
 # react-e [![CircleCI](https://circleci.com/gh/Originate/react-e.svg?style=svg)](https://circleci.com/gh/Originate/react-e)
 
-React helper to generate elements
+JSX replacement that doesn't need to be compiled.
 
-## Installing
-`npm install --save react-e`
+## Basic Usage
 
-## Example Usage
+```javascript
+e = require('react-e')
 
-```coffeescript
-e = require 'react-e'
-
-e '.container',
-  e 'button.primary', 'Click me'
-  e 'span.text', className: active: true, 'Some text'
+e(Modal,
+  e('.container',
```

Comment:
Woops the last comment ended up in the same thread, sorry about that.

In reply to @alexdavid, you have a responsibility to define the best use case in the readme, since most people will follow your example. I also think `$` is more intuitive. jQuery had a very good reputation, most people would love to use something like it again.